packages/core/contracts/optimistic-assertor/implementation/examples/DataAsserter.sol

```diff
@@ -23,40 +23,35 @@ contract DataAsserter {
         bool resolved; // Whether the assertion has been resolved.
     }
 
-    mapping(bytes32 => bytes32) public oaIdsToInternalIds;
-
     mapping(bytes32 => DataAssertion) public assertionsData;
 
-    event DataAsserted(bytes32 indexed dataId, bytes32 data, address indexed asserter, bytes32 oaAssertionId);
+    event DataAsserted(bytes32 indexed dataId, bytes32 data, address indexed asserter, bytes32 assertionId);
 
-    event DataAssertionResolved(bytes32 indexed dataId, bytes32 data, address indexed asserter, bytes32 oaAssertionId);
+    event DataAssertionResolved(bytes32 indexed dataId, bytes32 data, address indexed asserter, bytes32 assertionId);
 
     constructor(address _defaultCurrency, address _optimisticAssertor) {
         defaultCurrency = IERC20(_defaultCurrency);
         oa = OptimisticAssertorInterface(_optimisticAssertor);
         defaultIdentifier = oa.defaultIdentifier();
     }
 
-    // For a given dataId and asserter, returns a boolean indicating whether the data is accessible and the data itself.
-    function getData(bytes32 dataId, address asserter) public view returns (bool, bytes32) {
-        bytes32 dataAssertionId = getAssertionId(dataId, asserter);
-        if (!assertionsData[dataAssertionId].resolved) return (false, 0);
-        return (true, assertionsData[dataAssertionId].data);
+    // For a given assertionId, returns a boolean indicating whether the data is accessible and the data itself.
+    function getData(bytes32 assertionId) public view returns (bool, bytes32) {
+        if (!assertionsData[assertionId].resolved) return (false, 0);
+        return (true, assertionsData[assertionId].data);
     }
 
     // Asserts data for a specific dataId on behalf of an asserter address.
     function assertDataFor(
         bytes32 dataId,
         bytes32 data,
         address asserter
-    ) public returns (bytes32 oaAssertionId) {
-        bytes32 dataAssertionId = getAssertionId(dataId, asserter);
+    ) public returns (bytes32 assertionId) {
         asserter = asserter == address(0) ? msg.sender : asserter;
-        require(assertionsData[dataAssertionId].asserter == address(0), "Data already asserted");
```

Comment:
I understand there is no more use of this check since now the same asserted dataId/asserter would generate different `assertionId` unless its done in transactions that share the same `block.timestamp` as it is added to the `claim`. This affects how the contract works, so would also need to update block comment at the top.